Add back manual install content to mobile docs (#14343)
newrelic/docs-website: src/content/docs/mobile-monitoring/new-relic-mobile-cordova-phonegap/get-started-with-cordova-monitoring.mdx

import mobileQueryExample from 'images/mobile_screenshot-full_query-example.webp'

> Our New Relic Cordova agent monitors your Cordova mobile app and provides deep insights into your app's performance, errors, and user experience. Written in JavaScript, the Cordova agent includes all New Relic mobile monitoring features we offer for native mobile apps. Once you install and configure the Cordova agent, you'll be able to:

Reviewer: Is there a reason to mention the Javascript? If not I think we can cut it safely.

> Our New Relic Cordova agent monitors your Cordova mobile app and provides deep insights into your app's performance, errors, and user experience. Written in JavaScript, the Cordova agent includes all New Relic mobile monitoring features we offer for native mobile apps. Once you install and configure the Cordova agent, you'll be able to:

Author: ```suggestion
Our New Relic Cordova agent monitors your Cordova mobile app and provides deep insights into your app's performance, errors, and user experience. The agent includes all New Relic mobile monitoring features we offer for native mobile apps. Once you install and configure the Cordova agent, you'll be able to:
```

Before you install the Cordova agent, ensure your app meets these requirements:

* Cordova 7.x or higher
* Node.js 6.0 or higher
* Cordova CLI tools
* An [Android](https://cordova.apache.org/docs/en/latest/guide/platforms/android) or [iOS](https://cordova.apache.org/docs/en/latest/guide/platforms/ios/index.html) Cordova platform for your project
* Mobile [application tokens](/docs/mobile-monitoring/new-relic-mobile/maintenance/viewing-your-application-token)

## Install the Cordova agent [#install]

To install the Cordova agent, follow our [guided install](https://onenr.io/08jqaJD9pjl), located directly in the UI.

If you need to manually install for any reason, follow these steps

<Steps>
  <Step>

### Create an application token(s) [#app-token]

>   1. Go to **one.newrelic.com > Browser & Mobile > (choose Android or iOS)**, then complete the instrumentation steps in the UI. 

Reviewer: I think there's supposed to be a link to the platform here

>   1. Go to **one.newrelic.com > Browser & Mobile > (choose Android or iOS)**, then complete the instrumentation steps in the UI. 

Author: ```suggestion
  1. Go to **[one.newrelic.com](https://onenr.io/02wdd6ngXwE) > Browser & Mobile > (choose Android or iOS)**, then complete the instrumentation steps in the UI. 
```

  2. From **Step 4** in the UI, copy the application token(s) (one for iOS and Android if applicable).

</Step>
<Step>
### Add the Cordova agent [#install]

In your Cordova project directory, add the Cordova agent to your project using your command line tool, and paste the application token(s) in the `--variable` argument:

   ```
   # Install from github repository
   cordova plugin add https://github.com/newrelic/newrelic-cordova-plugin.git --variable IOS_APP_TOKEN="{ios-app-token}" --variable ANDROID_APP_TOKEN="{android-app-token}"
   ```

</Step>

<Step>

### (Optional) Configure the agent [#configure-agent]

To add optional configuration options, add them to the `--variable` argument, which will call them upon agent start. 

> The agent supports these configuration options:

Reviewer: I think you can join this sentence with the previous paragraph

> 
> To add optional configuration options, add them to the `--variable` argument, which will call them upon agent start. 
> 
> The agent supports these configuration options:

Author: ```suggestion

To add optional configuration options, add the following configurations to the `--variable` argument, which will call them upon agent start:
```
